src/pages/DudNew.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Enzyme, { shallow } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import NotFound from './NotFound'
+
+describe("When DudIndex renders", () => {
+    it("displays a form", () => {
+      const dudNew = shallow(<DudNew />)
```

Comment:
You do not need to repeat 
```javascript
const dudNew = shallow(<DudNew />)
```
Ror each time you call upon it. Delete the extras and I recommend formatting your test. 

Also lines 13 and 16 call on the variable label which is never declared. Double check that your tests are passing before you push up testing files!